baseball/src/Components/Display/Display.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import Dashboard from "../Dashboard/Dashboard";
+
+export default class Display extends Component {
+state = {
+    balls: 0,
+    strikes: 0
+};
+
+ballsHandler = e => {
+    e.preventDefault();
+    if (this.state.balls >= 4 || this.state.strikes >= 3) {
+    this.setState({
+        balls: 0,
+        strikes: 0
+    });
+    } else {
+    this.setState({
+        ...this.state,
+        balls: this.state.balls + 1
+    });
+    }
+};
+
```

Comment:
I think that based on the assignment and requirements, having separate handlers as you have is the best way to go 👍 Looking at your code made me think 🤔 of creating a sort of generic `pitchHandler` and then having a random `pitch event` generator to pass to that handler for a potential ball, strike, foul, or hit. Good stuff Stefan 🔥

Reply:
That would be cool Julian. I like to see how you randomize the pitchEvents.